lib/adf_web.py

```diff
@@ -684,7 +684,7 @@ def jinja_list(seas_list):
 
             #List of ADF default plot types
             avail_plot_types = res["default_ptypes"]
-            
+           
             #Check if current plot type is in ADF default.
             #If not, add it so the index.html file can include it
             for ptype in plot_types.keys():
```

Comment:
There was a problem causing ADF to crash in the for loop starting on line 687 in adf_web.py

I needed to change 

```
 for ptype in plot_types.keys():
                if ptype not in avail_plot_types:
                    avail_plot_types.append(plot_types)
                    
```
to


```
 for ptype in plot_types.keys():
                if ptype not in avail_plot_types:
                    avail_plot_types.append(ptype)
                    
```

Reply:
@adagj Thanks for catching this! Although this should've caught the `TimeSeries` plot type since it is included in `lib/adf_variable_defaults.yaml`. I'm curious why this crashed for you, would you be able to provide any further details when this crashed? Thanks!